src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/pact/PactContractTest.kt

```diff
@@ -77,6 +77,9 @@ class PactContractTest {
   @State("Referral can be submitted")
   fun `ensure referral can be submitted`() {}
 
+  @State("Referral summaries exist for an organisation with the ID a026cc07-8e0b-40dd-9b66-1b3dacecc63d")
+  fun `ensure referral summaries exist for an organisation with the ID a026cc07-8e0b-40dd-9b66-1b3dacecc63d`() {}
```

Comment:
I've also implemented this fix in my PR. Given it's to do with referral summaries it's probably best to axe this change in this PR; I'll merge mine in first so you can rebase.